_posts/2024-11-13-boosting-k-nn-exact-search.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Boosting k-NN exact search performance"
+authors:
+   - ryanbogan
+   - jmazane
+   - vamshin
+   - kolchfa
+date: 2024-11-13
+categories: 
+    - technical-posts
+has_science_table: true
+meta_keywords: exact k-NN search in OpenSearch, SIMD optimizations, query performance, relevance scoring, latency management
+meta_description: Learn how to achieve precise, high-performance search with exact k-NN in OpenSearch.
```

Comment:
Please update the meta with the following:

**meta _keywords:** k-NN search performance, SIMD in OpenSearch, script score queries, performance optimization, machine learning, exact K-NN

**meta_description:** Boost exact k-NN search performance in OpenSearch using SIMD optimizations and script_score queries. Discover real-world performance gains for efficient vector similarity searches in machine learning applications.

Reply:
Updated